config/wdio/wdio.conf.js

```diff
@@ -113,7 +116,10 @@ const config = {
     timeout: 1200000,
     bail,
   },
-
+  reporterOptions: {
+    outputDir: path.resolve(process.cwd(), 'tests', 'wdio', 'reports', 'terra-spec-results'),
```

Comment:
```suggestion
    outputDir: path.resolve(process.cwd(), 'tests', 'wdio', 'reports'),
```
If multiple reporters are used, I'd expect their results to be outputed under reports, not terr-spec-results